<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainform</class>
 <widget class="QWidget" name="mainform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>新宋体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="talk_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>对话</string>
   </property>
  </widget>
  <widget class="Line" name="dock_line1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="dock_line2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="sing_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>歌曲</string>
   </property>
  </widget>
  <widget class="Line" name="dock_line3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="person_button">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>形象</string>
   </property>
  </widget>
  <widget class="Line" name="dock_line4">
   <property name="geometry">
    <rect>
     <x>657</x>
     <y>30</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="setting_button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="Line" name="dock_line5">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>30</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="help_button">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>30</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
  <widget class="QPushButton" name="about_button">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>30</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </widget>
  <widget class="Line" name="mainform_line1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>1041</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>1161</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="statusbar_layout" stretch="0,0,0,4,0,0">
    <item>
     <widget class="QCheckBox" name="agreement_checkbox">
      <property name="text">
       <string>我已阅读并同意：</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="RainyFriend_agreement_button">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>《夜雨为伴用户使用协议》</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Apache_agreement_button">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>11</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>《Apache2.0开源协议》</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="status_bar_sp1">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="encourage_button">
      <property name="font">
       <font>
        <family>隶书</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>赞助</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="status_bar_sp2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>122</y>
     <width>1121</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="show_layout"/>
  </widget>
  <widget class="Line" name="mainform_line2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>660</y>
     <width>1101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Test_label">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>30</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>85</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>85</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>22</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>内测版本beta1</string>
   </property>
  </widget>
  <widget class="QLabel" name="back_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/mainback.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="back_label2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 215);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>back_label</zorder>
  <zorder>back_label2</zorder>
  <zorder>talk_button</zorder>
  <zorder>dock_line1</zorder>
  <zorder>dock_line2</zorder>
  <zorder>sing_button</zorder>
  <zorder>dock_line3</zorder>
  <zorder>person_button</zorder>
  <zorder>dock_line4</zorder>
  <zorder>setting_button</zorder>
  <zorder>dock_line5</zorder>
  <zorder>help_button</zorder>
  <zorder>about_button</zorder>
  <zorder>mainform_line1</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>horizontalLayoutWidget_2</zorder>
  <zorder>mainform_line2</zorder>
  <zorder>Test_label</zorder>
 </widget>
 <resources>
  <include location="NightRain.qrc"/>
 </resources>
 <connections/>
</ui>
